libraries/lib-concurrency/ParallelProgress.h

```diff
@@ -0,0 +1,55 @@
+/**********************************************************************
```

Comment:
No license identifier. Previously, Audacity ignored it in most files, but that is not correct.